### ISSUE TITLE
Jobrapido Fork

### ISSUE THREAD

**Issue by minobedin** [eyecatchup/php-webmaster-tools-downloads#24]
Hi, I'm trying to use fork made by Jobrapido for download data from Search Analytics. I'm trying to use only the gwtdata.php file from this fork, because I don't have the other Pre-Requisites like PDO library for PHP and XML library for PHP. Anyone use this fork currently? Does it work? When I call DownloadCSV function from my code with "SEARCH_ANALYTICS" value the scripts download only a html page from Google, but when i use "TOP_PAGES" or "TOP_QUERIES" it works great.

PS: in my code I fix the login problem removing the string "-php" in the gwtdata.php script. I suggest to Jobrapido to do the same thing in his fork.

**Comment by sblendorio** [comment]
You're right: the "SEARCH_ANALYTICS" case was under development and it has not been completed, for this reason it downloads just an HTML page.

Thanks for suggestion about the "-php" string to remove.
